src/foam/nanos/ruler/predicate/PropertyIsSetPredicate.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.ruler.predicate',
+  name: 'PropertyIsSetPredicate',
+  extends: 'foam.mlang.predicate.AbstractPredicate',
+  implements: ['foam.core.Serializable'],
+
+  documentation: 'A predicate that returns true when a specific property is set.',
+
+  javaImports: [
+    'foam.core.FObject',
+    'static foam.mlang.MLang.*'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'propName'
+    },
+    {
+      class: 'Boolean',
+      name: 'isNew',
```

Comment:
Please document the purpose of this property.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/3956/commits/7ece73e7bada754b5e6fdb6a44d8eaef5041c2a3.